.circleci/config.yml

```diff
@@ -24,6 +24,7 @@ env-testing-build: &env-testing-build
 
 env-release-build: &env-release-build
   GN_CONFIG: //electron/build/args/release.gn
+  STRIP_BINARIES: true
```

Comment:
this should be set on testing builds too

Reply:
@deepak1556 what do you think?

Reply:
I was thinking binaries used in CI testing can remain as such, since we tend to disable them at some point for debugging. Wouldn't it be okay if its just for release builds ?

Reply:
ah right, yes, ignore me. deepak is right. morning brain strikes again!

Reply:
@nornagon mind to approve then? )